### ISSUE TITLE
No styles after following Quick start

### ISSUE THREAD

**Issue by daleclements** [valor-software/ng2-handsontable#423]
Following the Quick start instructions:

I installed with "npm i ng2-handsontable --save" and didn't do step 2, because I'm using the CLI. Updated the module file per step 3, then added:

To the .html:
<hot-table 
  [data]="data"
  [options]="options">
</hot-table>

and to the component.ts:
  data: Array<any> = [
    [1,2,3],
    [4,5,6]
  ]
  options ={
    colHeaders: true,
    rowHeaders: true
  }

From the result it seems clear that no styles are being applied:
![image](https://cloud.githubusercontent.com/assets/3041062/25820552/a46d1ba2-33ff-11e7-97e1-7255f9fe36f5.png)

I've tried copying handsontable.css from the original non-angular handsontable project and including it in the component's styleUrls but that didn't work.

In the process of writing this I tried including it in a <link> tag in my index.html and that seems to have worked, but I'm still submitting to say that some mention of the required .css should be in the Quick Start section.

**Comment by ShaofeiZi** [comment]
were you able to solve this issue? I am having the same problem now.

**Comment by daleclements** [comment]
Yes but only in the way that I described above. I had to go to the original non-angular handsontable github page, copy out the handsontable.css file (https://github.com/handsontable/handsontable/blob/master/src/css/handsontable.css), and include it in a <script> tag in my index.html file. This doesn't feel like the right way to do it, but it's all I was able to do to get some styling.

**Comment by ShaofeiZi** [comment]
Thank you, but I still can not......

**Comment by jasonburrows** [comment]
@ShaofeiZi I took that css file, made a local copy, and referenced it directly within my master css file like this:  

`@import url('css/handsontable.css');`

I imagine you could add it to the `.angular-cli.json` file's `scripts` property if you are using the CLI, although I didn't test that.

It is working for me.  Certainly this step shouldn't be required, or should at least be documented.
